src/info/mod.rs

```diff
@@ -304,27 +304,58 @@ impl Info {
     }
 
     fn get_language_field(&self, title: &str) -> String {
-        let mut language_field = String::from("");
-
+        let language_bar_length = 26;
         let pad = title.len() + 2;
+        let color_palette = vec![
+            Color::Red,
+            Color::Green,
+            Color::Yellow,
+            Color::Blue,
+            Color::Magenta,
+            Color::Cyan,
+        ];
 
-        let languages: Vec<(String, f64)> = {
-            let mut iter = self.languages.iter().map(|x| (format!("{}", x.0), x.1));
+        let mut language_field = String::from("");
+        let mut i = 0;
+        let languages: Vec<(String, f64, Color)> = {
+            let mut iter = self.languages.iter().map(|x| {
+                i += 1;
+                let color = if self.config.true_color {
+                    x.0.get_colors(true)[0]
+                } else {
+                    color_palette[i % color_palette.len()]
+                };
+                (format!("{}", x.0), x.1, color)
+            });
             if self.languages.len() > 6 {
                 let mut languages = iter.by_ref().take(6).collect::<Vec<_>>();
                 let other_sum = iter.fold(0.0, |acc, x| acc + x.1);
-                languages.push(("Other".to_owned(), other_sum));
+                languages.push(("Other".to_owned(), other_sum, Color::White));
                 languages
             } else {
                 iter.collect()
             }
         };
 
+        for (_, language) in languages.iter().enumerate() {
```

Comment:
```suggestion
        for language in languages.iter() {
```
The index isn't used, so does `enumerate` have to be called?